Osiris/Hooks.cpp

```diff
@@ -324,6 +326,30 @@ static void __stdcall setDrawColor(int r, int g, int b, int a) noexcept
 static bool __stdcall fireEventClientSide(GameEvent* event) noexcept
 {
     if (event) {
+        if(!strcmp(event->getName(),"round_prestart") || !strcmp(event->getName(),"round_end"))
+        {
+            if (!strcmp(event->getName() ,"round_end"))
+            {
+
+                warmup = false;
+            }
+            else
+            {
+
+                warmup = true;
+            }
+
+        }
+
+        if (!strcmp(event->getName(), "round_freeze_end"))
+        {
+            warmup = false;
+        }
+
+        if (!strcmp(event->getName() ,"round_end"))
+        {
+            warmup = true;
+        }
```

Comment:
excessive whitespace

Reply:
this should all be handled in the switch case below, this adds unnecessary `strcmp`s